src/main/cpp/Traj.cpp

```diff
@@ -0,0 +1,44 @@
+#include "Traj.h"
```

Comment:
May I be a stickler about spelling things out?

```suggestion
#include "Trajectory.h"
```